app/config/publications/report-types.js

```diff
@@ -1,14 +1,96 @@
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+
 export default [
   {
-    translationKey: 'publication-reports--type--government-domain',
-    metricsTypeUri: 'government-domain', // TODO: these aren't URI's. that should change
+    translationKey: 'publication-reports--type--by-mandatees--on-decision-date',
+    metricsTypeUri: 'http://themis.vlaanderen.be/id/concept/publicatierapporttype/by-mandatees-on-decision-date', // TODO: these aren't URI's. that should change
+    // fixed job params for this report type
+    // TODO: should be set on the backend
+    fixedParams: {
```

Comment:
As discussed previously: ideally, these fixedParams are mapped in the backend based on the `metricsTypeUri`. However, we decided to leave this here for now, and pick this up in a separate story later.